### PR TITLE
Lower-case "mobile" everywhere it makes sense (DOC-6135)

### PR DESCRIPTION
**Pull request by austin-schaefer**
Compare to #2926 (downcasing Browser) where applicable)

### Notes for reviewer

- I've tested this locally, but I'd still appreciate if you can please pull down and test locally.
- While I changed doc/category names in frontmatter and *yml files, I didn't actually change any filenames or URLs. I felt like this is too risky to be worth the time given the minor changes, but please let me know if you disagree.
- The most important area to test is to poke around mobile taxonomy, menu, and intro docs. I made some taxonomy changes this PR and that's something I'm not very experienced with.
- While testing, I realized that the release notes now refer to an `XCFramework agent` rather than an `iOS agent`. I'm not planning to fix that with this PR, but I raised it as a question in the #mobile-instrumentation channel if you'd like to follow along. 

### To-dos

- [x] Check for instances of `new relic` in lower-case because why not
- [X] Check for instances of `New Relic Mobile`
- [x] Check for instances of `Mobile`
- [x] Check for instances of `Mobile Monitoring`
- [x] Try to fix this category https://docs.newrelic.com/docs/mobile-monitoring/new-relic-mobile/
- [x] And this category https://docs.newrelic.com/docs/mobile-monitoring/new-relic-mobile-android/
- [x] And this category https://docs.newrelic.com/docs/mobile-monitoring/new-relic-mobile-ios/
- [x] Retitle these docs
    - [x] Retitle `Introduction to New Relic Mobile for Android`
    - [x] Retitle `Upgrade New Relic Mobile's Android SDK`
    - [x] Retitle `Introduction to New Relic Mobile for iOS`
    - [x] Retitle `Upgrade New Relic Mobile's iOS SDK`
    - [x] Retitle `Upgrading New Relic Mobile's tvOS SDK`
    - [x] Retitle iOS compatibility doc
    - [x] Retitle Android compatibility doc

### Did not update

* "original pricing/billing" docs
* Anything to do with partner API
* Docs for deprecated features AMP or Unity
* Any doc in the legal/licenses category
* Anything code (attribute/event names, method names, etc)
* File names and file paths in docs repo (i.e., I didn't mess with changing URLs or nav structure, just the display names)

### CODE REVIEW

**Comment by austin-schaefer** [comment]
This PR deletes a very old doc on upgrading the agent to v4. Given that v4 was released in 2014, I feel confident deleting this doc. I'm attaching it to this PR for historical reference. 
[Upgrade New Relic Mobile's iOS SDK to v4.txt](https://github.com/newrelic/docs-website/files/6774173/Upgrade.New.Relic.Mobile.s.iOS.SDK.to.v4.txt)

File: src/content/docs/mobile-apps/new-relic-mobile-apps/authentication-alerts/alerting-new-relic-mobile-apps.mdx [hunk 1]
```diff
@@ -2,7 +2,7 @@
 title: Alerting with New Relic mobile apps
 tags:
   - Mobile apps
-  - New Relic Mobile apps
+  - New Relic m apps
```
Comment (paperclypse) [inline src/content/docs/mobile-apps/new-relic-mobile-apps/authentication-alerts/alerting-new-relic-mobile-apps.mdx:5]: pretty sure this should read "mobile apps"

File: src/content/docs/mobile-apps/new-relic-mobile-apps/tvos-app/introduction-apple-tv-app.mdx [hunk 2]
```diff
@@ -2,7 +2,7 @@
 title: Introduction to Apple TV mobile app
 tags:
   - Mobile apps
-  - New Relic Mobile apps
+  - New Relic m apps
```
Comment (paperclypse) [inline src/content/docs/mobile-apps/new-relic-mobile-apps/tvos-app/introduction-apple-tv-app.mdx:5]: mobile apps

File: src/content/docs/mobile-crash-rest-api-v1.mdx [hunk 3]
```diff
@@ -1,10 +1,10 @@
 ---
-title: Mobile Crash REST API v1
+title: mobile crash REST API v1
```
Comment (paperclypse) [inline src/content/docs/mobile-crash-rest-api-v1.mdx:2]: Should "mobile" be lowercase here?

File: src/content/docs/mobile-monitoring/mobile-monitoring-ui/crashes/handled-exceptions-occurrences.mdx [hunk 4]
```diff
@@ -4,13 +4,13 @@ tags:
...
 ---
 
-After you use New Relic Mobile's [**Handled exceptions** page](/docs/mobile-monitoring/mobile-monitoring-ui/crashes/handled-exceptions-analyze-trends-prevent-crashes) to explore patterns and trends, you can drill down into deeper details about specific exception occurrences. These can be [events](/docs/mobile-monitoring/mobile-monitoring-ui/crashes/introduction-mobile-handled-exceptions#hex-api) that New Relic monitors by default, or [custom events you created](/docs/mobile-monitoring/new-relic-mobile/maintenance/add-custom-data-new-relic-mobile#custom-events).
+After you [**handled exceptions** page](/docs/mobile-monitoring/mobile-monitoring-ui/crashes/handled-exceptions-analyze-trends-prevent-crashes) to explore patterns and trends, you can drill down into deeper details about specific exception occurrences. These can be [events](/docs/mobile-monitoring/mobile-monitoring-ui/crashes/introduction-mobile-handled-exceptions#hex-api) that New Relic monitors by default, or [custom events you created](/docs/mobile-monitoring/new-relic-mobile/maintenance/add-custom-data-new-relic-mobile#custom-events).
```
Comment (paperclypse) [inline src/content/docs/mobile-monitoring/mobile-monitoring-ui/crashes/handled-exceptions-occurrences.mdx:13]: "After you use handled exceptions"?

File: src/content/docs/mobile-monitoring/mobile-monitoring-ui/crashes/introduction-mobile-handled-exceptions.mdx [hunk 5]
```diff
@@ -46,7 +46,7 @@ By using handled exceptions with New Relic Mobile, you can identify and resolve
       </td>
 
       <td>
-        New Relic Mobile offers both an API and a user interface to explore the data most important to your business.
+        You can view exception data via API as well as UI:
```
Comment (paperclypse) [inline src/content/docs/mobile-monitoring/mobile-monitoring-ui/crashes/introduction-mobile-handled-exceptions.mdx:49]: "the UI"? or "in the UI"?

File: src/content/docs/mobile-monitoring/new-relic-mobile-android/get-started/new-relic-android-compatibility-requirements.mdx [hunk 6]
```diff
@@ -1,23 +1,23 @@
...
   - New Relic Mobile Android
   - Get started
 translate:
   - jp
-metaDescription: 'Compatible build systems, OS versions, and more for New Relic Mobile''s Android agent.'
+metaDescription: 'Compatible build systems, OS versions, and more for the Android agent.'
 ---
 
-New Relic Mobile is compliant with standard application stores, including Google Play Store, Amazon Appstore, and others. Before you [install and configure New Relic Mobile functionality](/docs/mobile-monitoring/new-relic-mobile-android/install-configure) with your Android applications, follow these guidelines for compatibility and other requirements.
+Mobile moniotring is compliant with standard application stores, including Google Play Store, Amazon Appstore, and others. Before you [install and configure the mobile agent](/docs/mobile-monitoring/new-relic-mobile-android/install-configure) with your Android applications, follow these guidelines for compatibility and other requirements.
```
Comment (paperclypse) [inline src/content/docs/mobile-monitoring/new-relic-mobile-android/get-started/new-relic-android-compatibility-requirements.mdx:12]: moniotring

File: src/content/docs/mobile-monitoring/new-relic-mobile-android/install-configure/configure-proguard-or-dexguard-android-apps.mdx [hunk 7]
```diff
@@ -32,7 +32,7 @@ To add support for [ProGuard](http://proguard.sourceforge.net/manual/usage.html)
    ```
 3. Clean and rebuild your project.
 4. Run your app in an emulator or device to start seeing data on your mobile app's [**Overview** page](/docs/mobile-apps/mobile-apps-dashboard).
-5. Optional: Use the New Relic Mobile **Settings** page to [customize your mobile app](/docs/mobile-apps/customizing-your-mobile-app-settings).
+5. Optional: Use the mobile monitorng **Settings** page to [customize your mobile app](/docs/mobile-apps/customizing-your-mobile-app-settings).
```
Comment (paperclypse) [inline src/content/docs/mobile-monitoring/new-relic-mobile-android/install-configure/configure-proguard-or-dexguard-android-apps.mdx:35]: monitorng

File: src/content/docs/mobile-monitoring/new-relic-mobile-ios/installation/upgrade-new-relic-mobiles-ios-sdk.mdx [hunk 8]
```diff
@@ -1,22 +1,29 @@
...
+  - /docs/mobile-monitoring/mobile-monitoring-installation/ios/upgrading-new-relic-mobiles-ios-sdk-v1-0
+  - /docs/mobile-apps/ios-1-upgrade
+  - /docs/mobile-monitoring/mobile-monitoring-installation/ios/upgrading-new-relic-mobiles-ios-sdk-v1
+  - /docs/mobile-monitoring/new-relic-mobile-ios/install-configure/upgrading-new-relic-mobiles-ios-sdk-v4
+  - /docs/mobile-monitoring/new-relic-mobile-ios/install-configure/upgrade-new-relic-mobiles-ios-sdk-v4
+  - /docs/mobile-monitoring/new-relic-mobile-ios/installation/upgrade-new-relic-mobiles-ios-sdk-v4/
 ---
 
-You must be the account Admin to install, configure, and upgrade New Relic Mobile functionality with your iOS applications. For information about the latest version, refer to the [release notes](/docs/releases/ios).
+You must be ah account Admin to install, configure, and upgrade the iOS agent. For information about the latest version, refer to the [release notes](/docs/releases/ios).
```
Comment (paperclypse) [inline src/content/docs/mobile-monitoring/new-relic-mobile-ios/installation/upgrade-new-relic-mobiles-ios-sdk.mdx:22]: should "ah" be "an"?

File: src/nav/full-stack-observability.yml [hunk 9]
```diff
@@ -2324,21 +2324,21 @@ pages:
                 path: /docs/mobile-monitoring/mobile-monitoring-ui/usage-pages/versions-analysis
               - title: Monthly uniques report
                 path: /docs/mobile-monitoring/mobile-monitoring-ui/usage-pages/monthly-uniques-report
-      - title: New Relic Mobile Android
+      - title: Android agent
```
Comment (paperclypse) [inline src/nav/full-stack-observability.yml:2327]: It's weird. This works in my local build, but gets a 404 in the Gatsby preview. I think probably Gatsby hasn't caught up to some of your changes.

**Review by paperclypse — APPROVED** [review]
I found some typos, but otherwise looks good.

**Comment by paperclypse** [comment]
@austin-schaefer This platform has a great feature: Doc titles aren't directly connected to URLs, like in Drupal. I think we should take advantage of this feature (like you are in this PR)!

Of course, if we're moving files around, rather than just renaming titles, then I think we want to add redirects.

(Other people may disagree with this stance, of course :)

File: src/nav/full-stack-observability.yml (hunk 9 above)
Comment (austin-schaefer) [reply src/nav/full-stack-observability.yml:2327]: Hmm, seems to be working for me now in the Gatsby Cloud. Maybe it just caught up? https://docswebsitedevelop-doc6135newrelicmobilecleanup.gtsb.io/docs/mobile-monitoring/new-relic-mobile-android/get-started/introduction-new-relic-mobile-android/

File: src/content/docs/mobile-apps/new-relic-mobile-apps/authentication-alerts/alerting-new-relic-mobile-apps.mdx (hunk 1 above)
Comment (austin-schaefer) [reply src/content/docs/mobile-apps/new-relic-mobile-apps/authentication-alerts/alerting-new-relic-mobile-apps.mdx:5]: Yikes, good catch

File: src/content/docs/mobile-crash-rest-api-v1.mdx (hunk 3 above)
Comment (austin-schaefer) [reply src/content/docs/mobile-crash-rest-api-v1.mdx:2]: Thanks

**Review by paperclypse — APPROVED** [review]
Looks good!

**Pull request merged**